cg/services/deliver_files/deliver_files_service/deliver_files_service_factory.py

```diff
@@ -104,10 +82,10 @@ def _get_file_fetcher(self, delivery_type: DataDelivery) -> FetchDeliveryFilesSe
 
     @staticmethod
     def _get_sample_file_formatter(
-        workflow: Workflow,
+        workflow: str,
     ) -> SampleFileFormatter | SampleFileConcatenationFormatter:
         """Get the file formatter service based on the workflow."""
-        if workflow in [Workflow.MICROSALT]:
+        if workflow in [Workflow.MICROSALT, "microbial-fastq"]:
             return SampleFileConcatenationFormatter(FastqConcatenationService())
         return SampleFileFormatter()
```

Comment:
this is not needed with my suggestion above

Reply:
implemented